public_html/administrator/components/com_jed/views/review/tmpl/edit.php

```diff
@@ -0,0 +1,159 @@
+<?php
+/**
+ * @package    JED
+ *
+ * @copyright  Copyright (C) 2005 - 2019 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
+
+/** @var JedViewReview $this */
+
+JHtml::_('behavior.formvalidator');
+JHtml::_('formbehavior.chosen', 'select');
+
+JFactory::getDocument()->addScriptDeclaration("
+	Joomla.submitbutton = function(task)
+	{
+		if (task == 'review.cancel' || document.formvalidator.isValid(document.getElementById('review-form')))
+		{
+			Joomla.submitform(task, document.getElementById('review-form'));
+		}
+	};
+");
+?>
+
+<form action="<?php echo JRoute::_('index.php?option=com_jed&view=review&layout=edit&id=' . (int) $this->item->id); ?>" method="post" name="adminForm" id="review-form" class="form-validate form-horizontal">
+    <fieldset>
+		<?php echo $this->form->renderField('published'); ?>
+    </fieldset>
+    <input type="hidden" name="task" value=""/>
+	<?php echo HTMLHelper::_('form.token'); ?>
+</form>
+
+<h3><?php echo Text::_('COM_JED_REVIEW_VIEW_EDIT_TITLE'); ?></h3>
+<table class="table table-striped" id="reviewList">
+    <tr>
+        <td><?php echo Text::_('COM_JED_REVIEWS_TITLE'); ?></td>
+        <td><?php echo $this->item->title; ?></td>
+    </tr>
+	<tr>
+		<td><?php echo Text::_('COM_JED_REVIEWS_SCORE'); ?></td>
+		<td>@TODO Overall Score Calculation</td>
+	</tr>
+	<tr>
+		<td><?php echo Text::_('COM_JED_REVIEWS_COMMENTS'); ?></td>
+		<td><?php echo $this->item->body; ?></td>
+	</tr>
+    <tr>
+        <td><?php echo Text::_('JGLOBAL_FIELD_CREATED_LABEL'); ?></td>
+        <td><?php echo HTMLHelper::_('date', $this->item->created_on, Text::_('COM_JED_DATETIME_FORMAT')) ?></td>
```

Comment:
Why not use the default date format in Joomla, I have used `DATE_FORMAT_LC6` in the extensions. Is there a particular reason to use a specific format for JED? If so, this needs to be changed in the extensions as well as we want this uniform across the extension.

Reply:
`DATE_FORMAT_LC6` - this is fine for backend, but I don't think Y-m-d H:i should be used on frontend. I didn't create `COM_JED_DATETIME_FORMAT` - it was already there so I went with it. It makes more sense to me. Can still change to DATE_FORMAT_LC6 if preferred.

Reply:
@JazParkyn Leave it as it is, I have updated my PR to use the JED datetime format. Either way is fine by me, as long as it is consistent.